contracts/token/ERC20WithPermit.sol

```diff
@@ -272,6 +294,9 @@ contract ERC20WithPermit is IERC20WithPermit, Ownable {
     ) private {
         require(sender != address(0), "Transfer from the zero address");
         require(recipient != address(0), "Transfer to the zero address");
+
+        _beforeTokenTransfer(sender, recipient, amount);
```

Comment:
Do we also want to add `_afterTokenTransfer`that OZ's ERC20 has just in case?

Reply:
It was added in 4.x from what I see and we can handle all our existing needs with just `beforeTokenTransfer` so I suggest we add it when it's really needed.